aiocache/decorators.py

```diff
@@ -55,7 +55,9 @@ def cached_decorator(func):
 
             try:
                 if await cache_instance.exists(cache_key):
-                    return await cache_instance.get(cache_key)
+                    result = await cache_instance.get(cache_key)
+                    await cache_instance.close()
```

Comment:
Hmmmm we don't need to block the user until the pool is closed. `asyncio.ensure_future` would be sufficient here